demisto_sdk/commands/common/tools.py

```diff
@@ -854,6 +854,10 @@ def pack_name_to_path(pack_name):
     return os.path.join(PACKS_DIR, pack_name)
 
 
+def pack_name_to_posix_path(pack_name):
```

Comment:
why are you specifically using `PosixPath` and not `Path`?

Reply:
The returned value inserted to `self._pkgs` (from the function `_get_packages`) is `List[PosixPath]` and not List[Path] as the type hint suggests.  Confirmed this with sdk's master version as well. Fixed the type hint where needed.